Lesson31/demo-app/Lesson31-merve2/src/shared-components/typography/BodyS/index.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import './styles.css';
+
+export const BodyS = () => {
```

Comment:
Same here, the component should take a prop as input, otherwise it's not possible to re-use it.